src/data/contributions.json

```diff
@@ -1,149 +1,68 @@
-[ 
+[
   {
     "author": "Jumboduck",
     "date": "05/2021",
-    "entries": [
-      284,
-      285,
-      287,
-      293
-    ]
+    "entries": [284, 285, 287, 293]
   },
   {
     "author": "NightlyHerb",
     "date": "04/2021",
-    "entries": [
-      261,
-      263
-    ]
+    "entries": [261, 263]
   },
   {
     "author": "Serqet",
     "date": "01/2021",
-    "entries": [
-      237
-    ]
+    "entries": [237]
   },
   {
     "author": "Jumboduck",
     "date": "12/2020",
-    "entries": [
-      233,
-      235
-    ]
+    "entries": [233, 235]
   },
   {
     "author": "FrozenEarth",
     "date": "07/2020",
-    "entries": [
-      205
-    ]
+    "entries": [205]
   },
   {
     "author": "Jumboduck",
     "date": "06/2020",
     "entries": [
-      157,
-      165,
-      169,
-      170,
-      171,
-      172,
-      173,
-      184,
-      185,
-      186,
-      188,
-      189,
-      190,
-      191,
-      193,
+      157, 165, 169, 170, 171, 172, 173, 184, 185, 186, 188, 189, 190, 191, 193,
       195
     ]
   },
   {
     "author": "Jumboduck",
     "date": "05/2020",
-    "entries": [
-      142,
-      144,
-      145,
-      146,
-      150,
-      151,
-      152,
-      155
-    ]
+    "entries": [142, 144, 145, 146, 150, 151, 152, 155]
   },
   {
     "author": "Neicigam",
     "date": "05/2020",
-    "entries": [
-      89,
-      91,
-      92,
-      98,
-      99,
-      102,
-      104,
-      106,
-      108,
-      109,
-      113
-    ]
+    "entries": [89, 91, 92, 98, 99, 102, 104, 106, 108, 109, 113]
   },
   {
     "author": "Serqet",
     "date": "05/2020",
-    "entries": [
-      123
-    ]
+    "entries": [123]
   },
   {
     "author": "Neicigam",
     "date": "04/2020",
     "entries": [
-      18,
-      20,
-      22,
-      23,
-      26,
-      27,
-      29,
-      30,
-      31,
-      32,
-      34,
-      35,
-      37,
-      38,
-      52,
-      53,
-      57,
-      58,
-      59,
-      60,
-      61,
-      62,
-      63,
-      64,
-      68,
-      74,
-      77,
-      78,
-      79,
-      80,
-      81,
-      83,
-      86,
-      88
+      18, 20, 22, 23, 26, 27, 29, 30, 31, 32, 34, 35, 37, 38, 52, 53, 57, 58,
+      59, 60, 61, 62, 63, 64, 68, 74, 77, 78, 79, 80, 81, 83, 86, 88
     ]
   },
   {
     "author": "Millo",
     "date": "09/2019",
-    "entries": [
-      11
-    ]
+    "entries": [11]
+  },
+  {
+    "author": "ArthisRoo",
+    "date": "07/2021",
+    "entries": [332]
   }
```

Comment:
The contributions are ordered most recent first in the file (to avoid having to scroll all the way down every time). Do you think you could put that entry at the very top?